uptane-standard.md

```diff
@@ -460,45 +470,45 @@ A repository's Root metadata distributes the public keys of the top-level Root,
 
-#### Metadata about Images {#targets_images_meta}
+#### Metadata about images {#targets_images_meta}
 
-To be available to install on clients, all images on the repository MUST have their metadata listed in a targets role.  Each targets role MAY provide a list of some images on the repository.  This list MUST provide, at a minimum, the following information about each image:
+To be available to install on clients, all images on the repository MUST have their metadata listed in a Targets role.  Each Targets role MAY provide a list of some images on the repository.  This list MUST provide, at a minimum, the following information about each image:
 
 * The image filename
 * The size of the image in bytes
 * One or more hashes of the image file, along with the hashing function used
 
 ##### Custom metadata about images
 
-In addition to the required metadata, Targets metadata files MAY contain extra metadata for images on the repository. This metadata can be customized for a particular use case. Examples of use cases for different types of custom metadata can be found in the Deployment Considerations document. However, there are a few important pieces of custom metadata that SHOULD be present in most implementations. In addition, there is one element in the custom metadata that MUST be present in the targets metadata from the director.
+In addition to the required metadata, Targets metadata files MAY contain extra metadata for images on the repository. This metadata can be customized for a particular use case. Examples of use cases for different types of custom metadata can be found in the Deployment Considerations document. However, there are a few important pieces of custom metadata that SHOULD be present in most implementations. In addition, there is one element in the custom metadata that MUST be present in the Targets metadata from the Director.
 
-Custom metadata also MAY contain a field or section that is demarcated as custom metadata that MUST match whenever two pieces of metadata are checked against each other--most commonly, when targets metadata from the Director and Image repositories are checked against each other.
+Custom metadata also MAY contain a field or section that is demarcated as custom metadata, and this field MUST match whenever two pieces of metadata are checked against each other. Most commonly, this occurs when targets metadata from the Director and Image repositories are checked against each other.
 
-The following information SHOULD be provided for each image on both the Image repository and the Director repository, and SHOULD be included in the "MUST match" section, if it is implemented:
+The following information SHOULD be provided for each image on both the Image repository and the Director repository and, if implemented, SHOULD be included in the "MUST match" section:
```

Comment:
I agree with the spirit of cleaning this up, but I think we ought to explicitly clarify that there are two different statements here: (1) this information SHOULD be provided, and (2) if the **_MUST match_** section is implemented, that is the correct place for this information. They way it reads (in both the previous version and in the proposed change), it sounds like we're saying "if you choose to provide this information, it SHOULD go in a MUST match section". That conflicts with the previous paragraph (because having a MUST match section at all is a MAY, not a SHOULD).

Reply:
Ok. Got it. So, let's  use what you have noted above: 

The information listed below SHOULD be provided for each image on both the Image repository and the Director repository. If  a "MUST match section" is to be implemented, that is where this information should be placed."

Reply:
Sounds good, except that the SHOULD should be capitalized.

Reply:
The second SHOULD? Ok, I can fix that.

Reply:
FIXED